draft-ietf-quic-transport.md

```diff
@@ -2169,6 +2173,9 @@ receiving a probe packet from a different address.  Servers MUST NOT send more
 than a minimum congestion window's worth of non-probing packets to the new
 address before path validation is complete.
 
+When a client wishes to migrate to a new local address that is of a different
+address family from the currently used server address, the client can attempt
```

Comment:
This can be simpler.  A client that migrates to a new address SHOULD use a preferred address from the same address family for the server. 

I wonder if we should forbid doing a new address lookup for the name or if we should stay silent on that point...

Reply:
+1 to the SHOULD.  

Let's not forbid lookups.  It's an interesting question whether other mechanisms of learning the server's addresses are useful or not; I think more deployment experience would be useful here.

Reply:
Agreed. I've incorporated @martinthomson's text. I agree with Jana that we shouldn't forbid lookups yet.

Reply:
I'm okay with the SHOULD.  The caveats on our network are:
- A lookup by address from a different network will lead you to a different machine.  Unless we then have it route back to the machine you were previously talking to, it will ignore / Reset you.  Of course, if it can't route, it shouldn't have the same Reset Token key, so you'll probably then drop the Reset....
- Using the SPA from a different network will lead you back to the same host, which might not be a good choice for you to keep using.  You'll likely get a prompt GOAWAY in HTTP/3.

Reply:
On our networks as well, lookups are unlikely to route to the same machine. Given that there is no reason to expect a lookup to successfully route to the same machine (even on the SAME network), forbidding it seems like the right choice.

Reply:
I think the DNS lookup issue is orthogonal to this PR. @martinthomson can you file a new issue to discuss it there?